### ISSUE TITLE
agent failed, code: 23

### ISSUE THREAD

**Issue by mueller-ma** [fossology/fossology#1148]
### Description

I tried to import code via svn, but the wget_agent job fails.

#### How to reproduce

1. Go to Upload => From VCS
2. Select SVN, enter https url to repo, username, password

### Versions

* Last commit id on master: On tag `3.3.0`
* Operating System (lsb_release -a):
````
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 9.5 (stretch)
Release:        9.5
Codename:       stretch
````

My Fossology setup:
Debian 9 running `Docker version 18.06.0-ce, build 0ffa825`.
Installed via:
`git clone https://github.com/fossology/fossology`
`cd fossology/install`
`docker-compose up`

#### Fossology logs

None:
````sh
[user@fossology ~/fossology (HEAD detached at 3.3.0)]$ docker exec -i -t install_fossology-scheduler_1 cat /var/log/fossology/*
cat: /var/log/fossology/*: No such file or directory
[user@fossology ~/fossology (HEAD detached at 3.3.0)]$ docker exec -i -t install_fossology-web_1 cat /var/log/fossology/*
cat: /var/log/fossology/*: No such file or directory

````

#### Apache logs

Logs generated under /var/log/apache2/error.log

#### Job logs

````
2018-07-26 13:43:44 wget_agent [0] :: JOB[2].wget_agent[174.localhost]: "FATAL wget_agent.c.521: [wget_agent.c:521] Error exit: 1, CMD:svn export https://svn.example.com/repo/foo  --username user --password password /srv/fossology/repository/ad494aad6b02/wget/wget.174.dir --no-auth-cache >/dev/null 2>&1"
2018-07-26 13:43:44 wget_agent [0] :: JOB[2].wget_agent[174.localhost]: "FATAL wget_agent.c.525: please make sure the URL of repo is correct, also add correct proxy for your version control system, command is:svn export https://svn.example.com/repo/foo  --username user --password password /srv/fossology/repository/ad494aad6b02/wget/wget.174.dir --no-auth-cache >/dev/null 2>&1, GlobalTempFile is:/srv/fossology/repository/ad494aad6b02/wget/wget.174, rc is:256. "
2018-07-26 13:43:44 wget_agent [0] :: JOB[2].wget_agent[174.localhost]: "FATAL main.c.243: upload 2 File retrieval failed: uploadpk=2 tempfile=/srv/fossology/repository/ad494aad6b02/wget/wget.174 URL=https://svn.example.com/repo/foo Type=SVN"
2018-07-26 13:43:44 wget_agent [0] :: JOB[2].wget_agent[174.localhost]: agent failed with error code 23
2018-07-26 13:43:44 wget_agent [0] :: JOB[2].wget_agent[174.localhost]: agent failed, code: 23
````

### Further information
I connected to the sheduler docker container and run the svn command from the Jobs log `svn export https://svn.example.com/repo/foo  --username user --password password /srv/fossology/repository/ad494aad6b02/wget/wget.174.dir --no-auth-cache`. It claims the issuer of the https certificate is invalid. How can I import our root certificate?

PS: I have seen #1076, but @kohldampfer claims that he can scan other repos on the server. I tried other, but all fail.

**Comment by mcjaeger** [comment]
maybe by using ca-certificates: https://www.brightbox.com/blog/2014/03/04/add-cacert-ubuntu-debian/

but you actually replaced the URL in your tests, right? (`https://svn.example.com/repo/foo`)

**Comment by mueller-ma** [comment]
> maybe by using ca-certificates: https://www.brightbox.com/blog/2014/03/04/add-cacert-ubuntu-debian/

AFAIK the ca-certificates are reset when I shutdown the docker container, right?

> but you actually replaced the URL in your tests, right? (https://svn.example.com/repo/foo)

I replaced the real url with example.com.

Instead of the docker installation I installed Fossology from source which is working fine. When I have some time over, I can set up a second installation for finding this issue, but right now I'm too busy for that. Closing this issue for now.
